test/passes/licm.wast

```diff
@@ -393,9 +394,34 @@
   (func $global
     (local $x i32)
     (loop $loop
+      ;; Moving this global.get out of the loop is ok. It is mutable, but there
+      ;; is no possible place where it can be set in the loop.
       (local.set $x (global.get $glob))
       (drop (local.get $x))
       (br_if $loop (local.get $x))
     )
   )
+  (func $global-call
+    (local $x i32)
+    (loop $loop
+      ;; As above, but now the loop does a call, which might in theory change
+      ;; mutable global state, so we cannot optimize.
+      (local.set $x (global.get $glob))
+      (call $global)
+      (drop (local.get $x))
+      (br_if $loop (local.get $x))
+    )
+  )
+  (func $global-call-immutable
+    (local $x i32)
+    (loop $loop
+      ;; As above, but now the global is immutable, so the call does not
+      ;; prevent us from optimizing.
+      ;; mutable global state.
```

Comment:
Is this last comment line left over from a copy-paste?

Reply:
Oops, yes, a copy-paste leftover...

Reply:
```suggestion
```